scss/linksAndSocials.scss

```diff
@@ -0,0 +1,89 @@
+.our-links {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 90%;
+  margin: 0 auto 40px auto;
+  color: var(--text-color-light);
+}
+
+.our-company {
+  display: flex;
+  flex-direction: column;
+}
+
+.gift-text {
+  color: var(--text-color-light);
+}
+
+.company-link {
+  text-decoration: none;
+  margin-bottom: 10px;
+  color: var(--text-color-light);
+}
+
+.our-links h4 {
+  margin-bottom: 10px;
+  color: var(--title-color);
+}
+
+.footer__content {
+  display: flex
+}
+
+.christmas-gift,
+.our-company,
+.our-services,
+.social {
+  margin-bottom: 25px;
+  transform: translateY(-50px);
+  opacity: 0;
+  transition: 0.8s;
+  transition-delay: 0.1s;
+}
+
+.our-company {
+  transition-delay: 0.2s;
+}
+
+.our-services {
+  transition-delay: 0.3s;
+}
+
+.social {
+  transition-delay: 0.4s;
+}
+
+.our-services {
+  display: flex;
+  flex-direction: column;
+}
+
+.services-link {
+  margin-bottom: 10px;
+}
+
+.social-link {
```

Comment:
The hovers don't work
The icons aren't centered
Spacing between elements should be done with gap

Reply:
Done